contract-tests/sse-contract-tests/src/event_outbox.cpp

```diff
@@ -37,13 +37,13 @@ void EventOutbox::do_shutdown(beast::error_code ec) {
 void EventOutbox::post_event(launchdarkly::sse::Event event) {
     auto http_request = build_request(callback_counter_++, std::move(event));
     outbox_.push(http_request);
-    flush_timer_.expires_from_now(boost::posix_time::milliseconds(0));
```

Comment:
API has been deprecated. The new API already exists in the oldest boost we support.